yarn.lock

```diff
@@ -872,10 +872,10 @@
   dependencies:
     react-native-vector-icons "^6.2.0"
 
-"@textile/react-native-sdk@1.2.5":
-  version "1.2.5"
-  resolved "https://registry.yarnpkg.com/@textile/react-native-sdk/-/react-native-sdk-1.2.5.tgz#c26f22a9d0dba3f5527a65d71eaa4a9df1f71e68"
-  integrity sha512-BjpObJszgMZLZNWuJEXrI1Z7EtovqCJsVpZUbVeR2ZL3IZlODpMXc+sDsfD79aIwjEJaB1PgMscHu3PFjSPOEA==
+"@textile/react-native-sdk@1.2.6":
+  version "1.2.6"
+  resolved "https://registry.yarnpkg.com/@textile/react-native-sdk/-/react-native-sdk-1.2.6.tgz#0b7472a401ab41dfe7cdf5d5faa30e98a36100a6"
+  integrity sha512-XkrGC5UwUJKuLoIwD1VIcEoeSkBYZpOeCs3s2115HMYyE8XL5v7LqV1uPGrv+etpf9dpxns+7beU9Ih7+0Wifw==
   dependencies:
     "@textile/js-types" "0.1.13-rc4"
```

Comment:
old types?

Reply:
Oh yea, forgot we can update those now

Reply:
Updated in the rn sdk layer and pulled that update in here